mvlearn/embed/kcca.py

```diff
@@ -111,53 +167,48 @@ def fit(self, Xs):
         """
         Xs = check_Xs(Xs, multiview=True)
 
-        components_ = kcca(
-            Xs,
-            self.reg,
-            self.n_components,
-            ktype=self.ktype,
-            sigma=self.sigma,
-            degree=self.degree,
-        )
+        x = Xs[0]
+        y = Xs[1]
 
-        self.weights_ = _listdot(Xs, components_)
+        N = len(x)
+        x = _center_norm(x)
+        y = _center_norm(y)
 
-        return self.weights_
+        self.Kx = _make_kernel(x, self.ktype, self.constant,
+                               self.degree, self.sigma)
+        self.Ky = _make_kernel(y, self.ktype, self.constant,
+                               self.degree, self.sigma)
 
```

Comment:
Since this inherits from BaseEmbed, shouldn't .fit() return self, so that you can call .fit_transform() from teh base class?